WordprocessingML/Wml2Ew/XsltTransformation.sql

```diff
@@ -19,6 +19,9 @@ DECLARE @TransformationVersion INT
 DECLARE @DuplicateToOtherLanguages INT
 	SET @DuplicateToOtherLanguages = $DuplicateToOtherLanguages$	-- Set to 1 if you want to copy the same definition to other languages
 
+DECLARE @DefaultItemGUID UNIQUEIDENTIFIER;
+	SET @DefaultItemGUID = N'$ItemGUID$'; --Fill ItemGUID from default eWay DB when manually deploying system word template on the server for the first time
```

Comment:
Parametr bych spíše nazval @ItemGUID `SET @ItemGUID = $ItemGUID$`. Výchozí hodnota by měla být NULL a vkládat tam bude jen pokud přijde paramtrem.

Komentář by měl spíš být `-- Fill GUID if you need to use the same identifier across multiple databases`